package howto;
option cc_generic_services = true;

message request_message {
    optional string name = 1;
}

message response_message {
    optional string hello = 1;
}

service hello_ssl_service {
    rpc send_block( request_message ) returns ( response_message );
}
